### ISSUE TITLE
Accessible Issue / Missing Aria-Label for input field

### ISSUE THREAD

**Issue by masteradhoc** [Automattic/WP-Job-Manager#2825]
**Describe the bug**
The following line seems to be missing an aria-label. Gets flagged by tools like https://wave.webaim.org/

https://github.com/Automattic/WP-Job-Manager/blob/fa43c7624b1901cb17089a0b6593d7bba8a6fa9c/templates/job-filters.php#L31

**Screenshots**
![image](https://github.com/Automattic/WP-Job-Manager/assets/6242098/653a3610-211b-43c3-86bd-1ef205c733d6)

**Expected behavior**
1) Field should have a aria-label
2) String should be translateable

**Comment by mikeyarce** [comment]
There are labels there for the fields, so this looks like a false positive.
https://github.com/Automattic/WP-Job-Manager/blob/fa43c7624b1901cb17089a0b6593d7bba8a6fa9c/templates/job-filters.php#L29-L32
